frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -211,3 +211,15 @@ export const GovtMasthead = ({
     </Box>
   )
 }
+
+// Used for rendering in app with fixed position
+export const AppMasthead = () => {
+  return (
+    <>
+      <Box height="2rem" width="100vw" position="fixed" top="0" zIndex="11">
+        <GovtMasthead />
+      </Box>
+      <Box height="2rem" width="100vw" />
+    </>
```

Comment:
I think this can be encapsulated in a single child instead of using fragments. Prefer non-fragments due to possibility of layout being messed up (i.e. parent expecting single child when using `AppMasthead`, but actually there were 2 children).

Should we also change the zIndex to an exposed constant `banner`? See https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/foundations/z-index.ts.

Try:

```tsx
<Box h="2rem" w="100vw" pos="relative">
  <Box h="2rem" w="100vw" pos="absolute" top="0" zIndex="banner">
    <GovtMasthead />
  </Box>
</Box>
```

Reply:
thanks! incorporated these changes